src/AbstractPathResolver.js

```diff
@@ -15,6 +15,14 @@ export default class AbstractPathResolver {
   }
 
   async extendContext(...contexts) {
+    if (contexts.length) {
+      const defaultLanguages = contexts.map(context => context && context['@context'] && context['@context']['@language']).filter(Boolean);
```

Comment:
Lot's of this can be syntactically cleaned up, e.g.
`const defaultLanguages = contexts.map(context => context?.['@context']?.['@language']).filter(Boolean);`